doc/source/_static/scipy.css

```diff
@@ -78,3 +78,8 @@ div.admonition>.admonition-title {
     gap: 10px;
     margin-bottom: 20px;
 }
+
+/* Wrap long titles in pages */
+h1 {
```

Comment:
Truncation happens in more than just the headers. It also happens in the "next"/"previous" links at the bottom of the page and in the heading list in the right sidebar (fortunately, the left sidebar appears to already support wrapping).

Reply:
Oh I did not notice that - thanks for bringing it up, I'll do a follow up.